### ISSUE TITLE
Multiple values for CalendarDate

### ISSUE THREAD

**Issue by CubaJas** [WickyNilliams/cally#26]
Great idea select multiple values for calendar date component.

**Comment by WickyNilliams** [comment]
you mean a new component like `<calendar-multi>` or something?

**Comment by CubaJas** [comment]
> you mean a new component like `<calendar-multi>` or something?

I think the prop `value` of the component <calendar-date> can be `string | string[]` type and the new prop `multiple` for detect if someone wants pass array of dates. If `multiple` doesn't pass to the component and value has an array, component should work with the first item of array.

Like this:
```html
<calendar-date
  value="arrayOfSelectedDays"
  multiple=""
  min="2024-01-01"
  max="2024-12-31"
  locale="en-GB"
>
```

**Comment by WickyNilliams** [comment]
I'll probably go for a new component tbh. They're different enough to warrant it, even if they share 90% of the code.

For context I started with range and single date being the same component, but they diverged enough over time that i split into individual components. I imagine the same would happen here eg maybe there's a a prop for max number of dates selectable, toggle events for when a day is selected/deselected, and so on

**Comment by WickyNilliams** [comment]
That said, like the idea! I'll do it

**Comment by WickyNilliams** [comment]
Just published `v0.6.0` which includes the `<calendar-multi>` component https://wicky.nillia.ms/cally/components/calendar-multi/
